### DISCUSSION TITLE
Using Robotframework Dryrun test status as status for auto merge?

### DISCUSSION THREAD

**Discussion by jairo1024** [renovatebot/renovate#7949] [Request Help]
### Which Renovate are you using?

23.80.0 version

Renovate Open Source CLI

WhiteSource Renovate App

Renovate Pro

### Which platform are you using?

GitHub Enterprise

### What would you like to do?

Hi this is my first time creating an issue on here. I currently have a private Robotframework repository that I would like to auto-merge on certain dependencies (requirements.txt). Every time a Pull request is created the Jenkins job for the PR will build a new docker image and run the `robot --dryrun` from a Makefile target. 

The only way that I am able to get the auto merge to work is by setting the configuration to `"requiredStatusChecks": null`, since it technically doesn't have any tests (But I would like the dryrun to pass before it gets auto merged). Can I use the `robot --dryrun` as my test status so renovate can then merge?

**Comment by viceice** [comment]
1. Which kind of renovate are you using? I'm sure you don't use multiple different renovate on one repo.

2. How do you configure Jenkins? It should report the succeed / faild build status back to github, do renovate can check it.

3. Renovate will at least 2 runs for automerge, because it assume that there are running status checks when pushing new commits. So you need to make sure to run renovate regularly, best is hourly.
